src/core/config/mem_limit/option.cpp

```diff
@@ -0,0 +1,15 @@
+#include "config/mem_limit/option.h"
+
+#include "config/names_and_descriptions.h"
+
+namespace config {
+using names::kMemLimitMB, descriptions::kDMemLimitMB;
+extern CommonOption<MemLimitMBType> const MemLimitMBOpt{
```

Comment:
Aggregate initialization could be more readable here

Reply:
I don’t fully understand, `CommonOption` is a class